tests/PhpParser/Node/NodeFactoryTest.php

```diff
@@ -39,16 +41,47 @@ public function testCreateArray(array $inputArray, Array_ $expectedArrayNode): v
      */
     public function provideDataForArray(): Iterator
```

Comment:
You can change the `@return` type to `Iterator<mixed>`, as the return data here are various :)